main.js

```diff
@@ -15,4 +19,119 @@ for (let i = 0; i < menuLinks.length; i += 1) {
+    languages: ['html', 'css', 'javaScript'],
+    liveBtn: 'https://principlebrothers.github.io/Portfolio-Mobile-Version/',
+    sourceBtn: 'https://github.com/principlebrothers/Portfolio-Mobile-Version/blob/main/index.html'
+  },
+  {
+    class: "Gymfit padding",
+    name: 'Tonic',
+    image: './images/Gymfit.png',
+    alt: 'Gymfit',
+    canopy: ['CANOPY', 'BackEndDev', '2022'],
+    description: 'A daily selection of privately personalized reads: no accounts or sign-ups required',
+    languages: ['html', 'css', 'javaScript'],
+    liveBtn: 'https://principlebrothers.github.io/Portfolio-Mobile-Version/',
+    sourceBtn: 'https://github.com/principlebrothers/Portfolio-Mobile-Version/blob/main/index.html'
+  },
+  {
+    class: "professional padding",
+    name: 'Multi-Post Stories',
+    image: './images/Professional.png',
+    alt: 'Professional',
+    canopy: ['CANOPY', 'BackEndDev', '2022'],
+    description: 'A daily selection of privately personalized reads: no accounts or sign-ups required',
+    languages: ['html', 'css', 'javaScript'],
+    liveBtn: 'https://principlebrothers.github.io/Portfolio-Mobile-Version/',
+    sourceBtn: 'https://github.com/principlebrothers/Portfolio-Mobile-Version/blob/main/index.html'
+  }
+];
+
+let len = projects.length;
+for (let i = 0; i < len; i += 1) {
+  const project = projects[i];
+
+  workSection.innerHTML += ` <div class="${project.class}">
+  <div class="frontpage-icon">
+      <img src="${project.image}" alt="${project.alt}">
+  </div>
+  <div class="tonic-wrapper">
+      <div class="tonic">
+          <h2>${project.name}</h2>
+          <ul class="tonic-section-header">
+              <li class="canopy">${project.canopy[0]}</li>
+              <li><img src="./images/Counter.png" alt="counter" class="counter" /></li>
+              <li class="back-end-dev">${project.canopy[1]}</li>
+              <li><img src="./images/Counter.png" alt="counter" /></li>
+              <li class="year">${project.canopy[2]}</li>
+          </ul>
+          <p class="tonic-details">${project.description}</p>
+      </div>
+
+      <ul class="programing-languages">
+          <li><a href="#">${project.languages[0]}</a></li>
+          <li><a href="#">${project.languages[1]}</a></li>
+          <li><a href="#">${project.languages[2]}</a></li>
+      </ul>
+      <button type="button" class="popup">See Project</button>
+  </div>
+</div>`;
+}
+
+const popUpWindow = document.querySelector('#popUp');
+const popUpButton = document.querySelectorAll('.popup');
+const overLay = document.getElementById("overlay");
+const liveButton = document.querySelector('.live');
+const sourceButton = document.querySelector('.source');
+
+for (let i = 0; i < popUpButton.length; i += 1) {
+  const btnSelected = popUpButton[i];
+  btnSelected.addEventListener('click', () => {
+    popUpWindow.style.display = 'block';
+    overLay.classList.add('active');
+    overLay.style.display = 'block';
+  })
+}
```

Comment:
- [ ] Kindly make sure when click in see project button, I can see the same project data not different one.
- [ ] Also consider extract image for desktop version and use it in model because the image for mobile version is stretched in desktop version
> If you need more explanation, you can check this [loom video](https://www.loom.com/share/0a2aea8ee9e547f2adf0a75e632b8ec2)
>

Reply:
@omarsalem7 I appreciate the feedback given. It has given me a tough time but I have learned. With regard to using the desktop images. It will mean I have to change almost all the projects so I have to maintain the mobile version images.